src/EditorFeatures/CSharpTest/ExtractMethod/ExtractMethodTests.cs

```diff
@@ -9188,13 +9188,13 @@ private static K NewMethod()
         public async Task Script_ArgumentException()
         {
             var code = @"using System;
-public static void GetNonVirtualMethod<TDelegate>( Type type, string name)
+static void GetNonVirtualMethod<TDelegate>( Type type, string name)
```

Comment:
please don't update test cases.  instead still record the change in behavior for this case, and add a new test for the differnet case you carea bout.

Reply:
to be honest, I do not know why I even touched this method